authors/filip-srdoc/15-porezi/hr_HR/final.md

```diff
@@ -0,0 +1,107 @@
+Prema [FATF-u][fatf] (Financial Action Task Force) virtualni novac obilježavaju klasične funkcije novca koje kažu da je novac, pa tako i onaj virtualni, sredstvo razmijene, mjera vrijednosti te služi za pohranu vrijednosti. No, postoji razlika između klasičnog [fiat][fiat] novca i virtualnog novca. Naime, fiat novac je službeno sredstvo plaćanja u nekoj državi kojeg izdaje centralna banka te države za razliku od virtualnog novca koji nigdje nije prihvaćen kao službeno sredstvo plaćanja i koji nastaje rudarenjem o kojem više možete doznati [ovdje][bc].
+
+Virutalni se novac se razlikuje od elektroničkog novca koji predstavlja digitalnu inačicu prave fiat valute, no ta digitalna valuta može predstavljati i virtualnu i fiat valutu. Najpoznatija digitalna valuta na svijetu danas je bitcoin. Budući da smo o tehničkoj strani bitcoina pisali [ovdje][btc], u ovom članku ćemo se posvetiti porezno-pravnom aspektu.
+
+Bitcoin, kao i većina drugih [kriptovaluta][cc], se može steći rudarenjem, kupnjom na [mjenjačnicama][exc] tj. burzama kriptovaluta ili nasljeđivanjem. U nastavku ćemo govoriti o izravnim i neizravnim porezima na transakcije kriptovaluta, no za početak idemo vidjeti koja su temeljna načela za određivanje načina oporezivanja transakcija neke kriptovalute.
+
+## Temeljna načela za određivanje načina oporezivanja transakcija kriptovaluta
+
+[Što je kriptovaluta?][cc] To znamo manje-više svi, no što je kriptovaluta u očima države? Roba, novac, vrijednosti papir ili nešto treće? Kako bismo odredili izravne i neizravne poreze na kriptovalute, prvo ih moramo klasificirati. Neke države na kriptovalute gledaju kao na dobra odnosno _asset_, dok nekim drugima kriptovaluta predstavlja novac.
+
+Da bi, primjerice, bitcoin bio novac on mora zadovoljavati tri funkcije s početka članka. S obzirom da ga ljudi koriste u transakcijama, prva funkcija sredstva razmjene je zadovoljena. No, kad govorimo o drugoj, funkciji mjere vrijednosti, oko nje se neće svi složiti. Naime, da bi nešto moglo biti jedinica za mjeru vrijednosti ono samo mora imati neku lako odredljivu intrinzičnu vrijednost. Tu se postavlja pitanje: što određuje vrijednost kriptovalute? Vrijednost kriptovalute izražavamo drugim svjetskim priznatim valutama što nas dovodi do zaključka da bez poznavanja tečaja neke kriptovalute u odnosu na neku svjetski priznatu valutu ne možemo odrediti njezinu vrijednost. Treća funkcija virtualnog novca odnosno valute govori da valuta treba biti pouzdana i stabilna kako bi služila za pohranu vrijednosti. Iako na bitcoin i ostale kriptovalute ne utječe politička situacija niti [inflacija][inf] i slični problemi koji stvaraju probleme fiat valutama, postoji mogućnost da na vrijednost jedne kriptovalute utječe druga, jača kriptovaluta. Naravno, svi ovi uvjeti mogu, a i ne moraju biti ostvareni u budućnosti.
```

Comment:
- koriste u transakcijama ([hipotetski] - link na digital gold članak)
- svjetski priznatim
- ne utječe politička situacija (barem tako tvrdi RRIF, mi se s time [ne slažemo] - link na neki post tipa digital gold, ili tweets that crash markets ono kad je charlie lee tweetao i srušio tržište)

Reply:
ovi linkovi nemaju bas neke veze s politickom situacijom na koju se ovdje misli (ratovi, promjena vlade i sl.)